cost/superseded_instance/superseded_instance.pt

```diff
@@ -413,15 +413,42 @@ policy "pol_aws_superseded_instance" do
     detail_template <<-EOS
 # Superseded Instances
 
-| Resource UID | Name | Current Instance Type | Region | New Instance Type | VPC Required | ENA Required | Monthly Estimated Cost |
-| ------------ | ---- | --------------------- | ------ | ----------------- | ------------ | ------------ | ---------------------- |
-{{ range data -}}
-| {{ if .warn }}**_{{ end -}}{{ .resource_id }}{{ if .warn }}_**{{ end -}} | {{ .name }} | {{.instance_type}} | {{.region}} | {{.new_instance_type}} | {{ if .vpc_incompatible_move }} **_TRUE_** {{else}} false {{end}} |  {{ if .ena_incompatible }} **_TRUE_** {{else}} false {{end}} | {{.run_rate}} |
-{{ end -}}
-
 ___
 ###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
 EOS
+    export do
+      field "if__warn" do
+        label "Resource UID"
+        path jmes_path(item, "if .warn")
+        format "printf(%s**_ end -}} .resource_id }} if .warn }}_** end -}})"
```

Comment:
Ideally we might do something like
```
field "resource_uid" do
  label "Resource UID"
  format switch(val(item, "warn"), "bold", "")
end
```

But the "format" field currently doesn't accept functions -- might change this on the backend to support this, though it currently doesn't work like this.